src/components/CardBody.js

```diff
@@ -0,0 +1,133 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+import companyIcon from '../icon/monitor.svg'
+import countryIcon from '../icon/location.svg'
+import emailIcon from '../icon/letter.svg'
+import capstoneIcon from '../icon/cube.svg'
+import personalWebsiteIcon from '../icon/world.svg'
+import githubIcon from '../icon/github.svg'
+import codepenIcon from '../icon/codepen.svg'
+
+export default function CardBody({
+  company,
+  country,
+  email,
+  capstoneName,
+  capstoneLink,
+  personalWebsite,
+  github,
+  codepen,
+  skills,
+}) {
+  return (
+    <StyledBody>
+      <hr />
+      <ul>
+        <li>
+          <Icons src={companyIcon} alt="icon"></Icons>
```

Comment:
Maybe name the styled component IconsStyled ? If the icons are added via a `<span>` or `<i>` tag in the styled components, you could get rid of the img alt text :) Otherwise, maybe change the alt text to a descriptive text?